specs/codecs/dag-eth/convenience_types.md

```diff
@@ -65,7 +62,7 @@ the sets of transactions and receipts for that block in order to avoid the need
 and receipt tries to collect these sets (as is required when starting from a canonical `Header` block).
 These objects can be generated or verified by following the links within the contained `Header` to collect the `Transactions` and `Receipts`
 from the referenced transaction and receipt tries.
-* The IPLD block is a CBOR serialization of the object
+* The IPLD block is a CBOR serialization of the Block object.
```

Comment:
`Block` has links in it, so this should probably be `DAG-CBOR` I think?

```suggestion
* The IPLD block is a DAG-CBOR serialization of the `Block` object.
```

Reply:
Somewhat confused by this actually -- are CBOR objects present in Ethereum already?  Or are CBOR objects being synthesized here?

Reply:
If being synthesized -- why does their codec matter, what does that come from?  Did they need to get a CID generated for some reason?